server/appServices.js

```diff
@@ -282,6 +306,64 @@ async function watchMobileSecurityApps(namespace, kubeclient) {
   });
 }
 
+function handleUPSEventData(event, suffix, namespace, kubeclient) {
+  if (event.type === 'ADDED' && event.object && event.object.metadata.name.endsWith(suffix) && !updating) {
+    const data = getApp(appCollect, getAppName(event));
+
+    const currentServices = data.status.services.filter(ser => ser.name === 'push');
+    const service = currentServices[0];
```

Comment:
`const currentServices = data.status.services.filter(ser => ser.name === 'push');` can be removed:

```suggestion
    const service = data.status.services.find(ser => ser.name === 'push');
```

Reply:
Right now it could be removed. But may cause is we are planning to allow users have their customs services in MDC. Which if my thinking is right we can't guarantee that the `push` service would be the first in the list hence the reason to filter.

Reply:
Not sure I get you. With 

`data.status.services.find(ser => ser.name === 'push');`

and with the previous

`data.status.services.filter(ser => ser.name === 'push');`

We are filtering to get **only** push service.

Reply:
Sorry I miss read suggestion, I will change `.filter` to be `.find`